monochart/util/labels-update/check-if-already-updated.bash

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+##############################################################
+# This script checks if a deployment's labels have already
+# been updated.
+#
+# The following positional arguments are required:
+#   - RELEASE - e.g. monochart-testing
+#   - RELEASE_NAMESPACE - e.g. staging
+##############################################################
+
+set -e
+
+message() {
+  echo
+  echo "$1"
+  echo
+}
+
+RELEASE_NAME=$1
+RELEASE_NAMESPACE=$2
+
+# Check if the required arguments are present
+if [[ -z "$RELEASE_NAME" ]] || [[ -z "$RELEASE_NAMESPACE" ]]; then
+  message '⚠️  Missing required arguments. Please provide the following:'
+  message '    - RELEASE_NAME - e.g. monochart-testing'
+  message '    - RELEASE_NAMESPACE - e.g. staging'
+  exit 1
+fi
+
+helm_values=$(helm get values -n "$RELEASE_NAMESPACE" "$RELEASE_NAME")
+DEPLOYMENT=$(yq eval .fullnameOverride <<< "$helm_values")
+if [[ "$DEPLOYMENT" == "null" ]]; then
+  message "⚠️  Could not determine deployment name. Exiting."
+  exit 1
+fi
+
+echo "***********************************"
+echo "RELEASE_NAMESPACE: ${RELEASE_NAMESPACE}"
+echo "RELEASE_NAME: ${RELEASE_NAME}"
+echo "DEPLOYMENT: ${DEPLOYMENT}"
+echo "***********************************"
+
+deployment_yaml=$(kubectl get deployment -n "$RELEASE_NAMESPACE" "$DEPLOYMENT" -o yaml 2>&1)
+if [[ "$?" == 1 ]]; then
+  if [[ "$deployment_yaml" =~ "not found" ]]; then
+    echo "⚠️  The deployment ${DEPLOYMENT} does not exist."
+    exit 1
+  else
+    echo "❗ There was an error checking the deployment: $deployment_yaml"
+    exit 1
+  fi
+fi
+
+label_app_name=$(yq eval .spec.template.metadata.labels.\"app.kubernetes.io/name\" <<< "$deployment_yaml")
+if [[ "$label_app_name" == "$DEPLOYMENT" ]]; then
+  echo "Already updated."
+else
+  echo "Needs updating."
+fi
```

Comment:
We may want an exit code here? Whatever is easiest to check from the pipeline.

Reply:
+1 agree

Reply:
do both - the text is for humans, and the exit code is for computers.

Reply:
bump, we need an exit code to decide on firing off the script while deploying🤞

Reply:
```suggestion
if [[ "$label_app_name" == "$DEPLOYMENT" ]]; then
  echo "Already updated."
  exit 0
else
  echo "Needs updating."
  exit 1
fi
```

Reply:
> it never got to the `test` echo

@kos-kamil - can you use the output `Needs updating.`?